stock_orderpoint_manual_procurement_uom/wizards/make_procurement_orderpoint.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, _
+from odoo.exceptions import UserError, ValidationError
+
+
+class MakeProcurementOrderpoint(models.TransientModel):
+    _inherit = 'make.procurement.orderpoint'
+
+    @api.model
+    def _prepare_item(self, orderpoint):
+        vals = super(MakeProcurementOrderpoint, self)._prepare_item(orderpoint)
+        if orderpoint.procure_uom_id:
+            product_uom = orderpoint.procure_uom_id
+            vals['uom_id'] = product_uom.id
+        return vals
+
+    @api.multi
+    def make_procurement(self):
```

Comment:
This is a full override of the method from stock_orderpoint_manual_procurement without using super (https://github.com/OCA/stock-logistics-warehouse/pull/385/files#diff-caa1ed47e650d4d52dc2c98f85bee472R47).
Please extract the arguments passed to the `run` method in a `_prepare_procurement_group` method in the parent addon in https://github.com/camptocamp/qoqa_openerp/blob/master/odoo/local-src/speedy_views/models/procurement.py#L15-L22 so it allows you to extend this new method here and change  the quantity with _compute_quantity without replacing the parent method.

Reply:
@guewen Second link gives me "Page not found"

Reply:
@guewen I have removed `make_procurement()` method and set logic in `_prepare_procurement_values`.